debugger/debugger.jsx

```diff
@@ -30,14 +30,22 @@ const items = [
         platform: 'browser',
     },
 ]
-
-let initialState = new URL(window.location).searchParams.get('state')
-if (!keys.includes(initialState)) {
+let searchParams = new URL(window.location.href).searchParams
+let initialState = searchParams.get('state')
+if (!initialState || !keys.includes(initialState)) {
     initialState = 'protectionsOn_blocked'
 }
 
+let reflectList = ['screen']
+let reflectParams = new URLSearchParams(Object.entries({ state: initialState }))
+for (let [key, value] of searchParams) {
+    if (reflectList.includes(key)) {
+        reflectParams.set(key, value)
+    }
+}
+
```

Comment:
This represents a set of keys that might be set on the outer debugger iframe. Anything in `reflectList` should be forwarded to each iframe instance